cusub_common:
  waypoint_navigator:
    carrot_drive: 3.0